src/qibolab/unrolling.py

```diff
@@ -19,6 +19,7 @@ def _waveform(sequence: PulseSequence):
 
 
 def _readout(sequence: PulseSequence):
+    # TODO: Do we count 1 readout per pulse or 1 readout per multiplexed readout ?
```

Comment:
It's a memory issue, so it should be 1 per readout, because you will need memory to store the readout for each qubit.

Reply:
Could be, I am not sure about the behavior when they calculate and store a multiplexed pulse, would they need to store each readout as well ?

Reply:
Well, think about discrimination: you want to have one bit per-qubit, isn't it?

And with waveforms should be the same: unless in specific cases, it's not very meaningful to aggregate the information from different qubits. Isn't it?